VideoWeb/VideoWeb/ClientApp/src/app/shared/privacy-policy/privacy-policy.component.ts

```diff
@@ -2,8 +2,7 @@ import { Component, OnInit } from '@angular/core';
 
 @Component({
     selector: 'app-privacy-policy',
-    templateUrl: './privacy-policy.component.html',
-    styleUrls: ['./privacy-policy.component.css']
+    templateUrl: './privacy-policy.component.html'
```

Comment:
@shaed-parkar is the removing the styleUrls css deliberate?

Reply:
Yes, the stylesheet was used to hide elements in print but the GovUK stylesheet update means we can use `govuk-!-display-none-print` hide elements rather than use our own custom styles. This change meant the stylesheet became empty